utils/lib.sh

```diff
@@ -725,6 +725,16 @@ pyenv.cmd() {
     )
 }
 
+
+pyenv.activate() {
+	pyenv.install
+    # shellcheck source=/dev/null
+	source "${PY_ENV_BIN}/activate"
```

Comment:
nit: mixed space and tabs

Reply:
I've fixed locations outside the pyenv.activation scope.